### DISCUSSION TITLE
Easiest way to move wallets and devices to new computer

### DISCUSSION THREAD

**Discussion by freddiemacd** [cryptoadvance/specter-desktop#1026] [Help]
Can you just install specter on the new computer and copy over the .specter folder?

Or do you need to import each backup file separately?

**Comment by ben-kaufman** [comment]
Hi, the easiest way would be going to the Settings, General tab, then click there download backup. This will download your backup files. Then on the new Specter, go also to Settings General tab and click restore, extract first the zip into a folder, then select that folder to restore from.
